src/components/mainComponents/MainSliderSection/MainSliderSection.module.scss

```
@@ -0,0 +1,93 @@
+@import "../../../style/mixins";
+
+.sliderSection{
```

```suggestion
.sliderSection {
```
prettier для стилей надо настроить